src/components/entity/ha-state-label-badge.html

```diff
@@ -10,19 +10,19 @@
       }
 
       ha-label-badge {
-        --ha-label-badge-color: rgb(223, 76, 30);
+        --ha-label-badge-color: var(--label-red);
```

Comment:
Done. Also added the old colour (html-ified it to #DF4C1E) as second option. Furthermore added a special line that explicitly defined class .red as red, and added .yellow badge colour. Currently not used but can be in the future.